### PR TITLE
NFDIV 1672 clean up & NFDIV 1781

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-1672
https://tools.hmcts.net/jira/browse/NFDIV-1781

### Change description ###
image
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/steps/applicant1/review-the-application/content.ts [hunk 1]
```diff
@@ -144,7 +131,7 @@ const en = ({ isDivorce, userCase, partner, userEmail, isApplicant2 }: CommonCon
     userCase.applicant2SolicitorFirmName ? userCase.applicant2SolicitorFirmName : 'not given'
   }`,
   solAddress: `Solicitor address: ${
-    userCase.applicant2SolicitorAddress ? userCase.applicant2SolicitorAddress : 'not given'
+    userCase.applicant2SolicitorAddress?.trim() ? userCase.applicant2SolicitorAddress : 'not given'
```
Comment (lara-mpt) [inline src/main/steps/applicant1/review-the-application/content.ts:134]: does this _userCase.applicant2SolicitorAddress_ not need to be joined with the other fields like below? 

i.e 
       [
          userCase.applicant2SolicitorAddress1,
          userCase.applicant2SolicitorAddress2,
          userCase.applicant2SolicitorAddress3,
          userCase.applicant2SolicitorAddressTown,
          userCase.applicant2SolicitorAddressCounty,
          userCase.applicant2SolicitorAddressPostcode,
          userCase.applicant2SolicitorAddressCountry,
        ]
      .filter(Boolean)
      .join('\<br>\');

Comment (velimd) [reply src/main/steps/applicant1/review-the-application/content.ts:134]: if you mean data wise, then no. applicant2SolicitorAddress has the info, the FE solicitor address is multiple input boxes but the backend/xui is a string/textarea so when we post the info, its converted to a string with new line e.g. test\npostcode\ncountry and stored in applicant2SolicitorAddress.
if you mean format wise, James is away and there hasnt been an indication on how it should look, i did show it to Michael and he was okay with it but i think he is looking at it from a content wise. I did change it for the repondent address format as i think it should be formated with new lines.

**Review by lara-mpt — APPROVED** [review]

**Review by linusnorton — APPROVED** [review]

File: src/main/steps/applicant1/review-the-application/content.ts [hunk 2]
```diff
@@ -11,7 +11,22 @@ import { enConnectionBulletPointsSummarisedForAllUsers } from '../../../app/juri
 import { jurisdictionMoreDetailsContent } from '../../../app/jurisdiction/moreDetailsContent';
 import { CommonContent } from '../../common/common.content';
 
-const en = ({ isDivorce, userCase, partner, userEmail, isApplicant2, isJointApplication }: CommonContent) => ({
+const displayCorrectAddress = (applicant: 'applicant1' | 'applicant2', userCase: Partial<CaseWithId>) => {
+  const isSolicitorsAddress = `${applicant}${userCase[`${applicant}SolicitorAddress`]?.trim() ? 'Solicitor' : ''}`;
```
Comment (benouaer) [inline src/main/steps/applicant1/review-the-application/content.ts:15]: The name suggests it's a Boolean, but it's a string right

Comment (velimd) [reply src/main/steps/applicant1/review-the-application/content.ts:15]: yes its a string, i dont think it suggests a boolean. if you think so what name would you prefer?

Comment (linusnorton) [reply src/main/steps/applicant1/review-the-application/content.ts:15]: isXYZ kind of suggests boolean, maybe remove the is

Comment (velimd) [reply src/main/steps/applicant1/review-the-application/content.ts:15]: ah my bad, i was looking at displayCorrectAddress

**Review by benouaer — APPROVED** [review]

**Pull request merged**